components/feature/tab-collections/src/main/java/mozilla/components/feature/tab/collections/TabCollectionStorage.kt

```diff
@@ -23,6 +23,7 @@ import java.util.UUID
 /**
  * A storage implementation that saves snapshots of tabs / sessions in named collections.
  */
+@Suppress("TooManyFunctions")
```

Comment:
Will put up a PR to increase threshold for this. We have lots of these suppresses for classes that are perfectly fine, imo.